src/nodes/appServiceNodes.ts

```diff
@@ -83,8 +83,6 @@ export class AppServiceNode extends NodeBase {
         
         return [
             new DeploymentSlotsNode(this.site, this.subscription),
-            new FilesNode('Files', '/site/wwwroot', this.site, this.subscription),
```

Comment:
Please just comment this code out instead of deleting it. Also create an issue to re-enable this functionality and add a link to the issue in the comment

The issue should explain why we're disabling the functionality and what we need to do before we can re-enable it

Reply:
#45